src/ol_dbt/models/marts/combined/marts__combined_course_enrollment_detail.sql

```diff
@@ -185,6 +187,7 @@ with combined_enrollments as (
         , combined_enrollments.courserungrade_is_passing
         , combined_courseruns.course_title
         , combined_courseruns.course_readable_id
+        , null as courserun_upgrade_deadline
```

Comment:
Nitpick. Since this field is already set in `int__combined__courserun_enrollments`, it can just be like
```suggestion
         , combined_enrollments.courserun_upgrade_deadline
```